### DISCUSSION TITLE
Add support for italics in the theme.

### DISCUSSION THREAD

**Discussion by khatterr** [primer/github-vscode-theme#289] [Ideas]
Many themes in the marketplace have italics support that makes a life a lot easier, especially in front end development. 

For eg:-  When the HTML markup is a large file then if the attributes are italicised, it would be a lot easier to look at and modify the markup.

Please provide support for this feature. It would be a life saver.

Thank You

**Comment by simurai** [comment]
Do you mean making HTML attributes italic, like in this example?

![Screen Shot 2022-07-13 at 17 06 55](https://user-images.githubusercontent.com/378023/178683818-5b815326-0cb5-4e88-8035-8e234e20ef95.png)

Hmm... in general, this theme tries to stay close to how github.com highlights HTML attributes which are non-italic. But we can convert this issue to a discussion and see if we get more feedback.

**Comment by simurai** [comment]
Based on this [comment](https://stackoverflow.com/a/58407103), you can also enable this in your `settings.json` with:

```json
"editor.tokenColorCustomizations": {
    "textMateRules": [
        {
            "scope": "entity.other.attribute-name",
            "settings": {
                "fontStyle": "italic"
            }
        }
    ]
}
```

**Comment by simurai** [comment]
Moving from https://github.com/primer/github-vscode-theme/issues/213 where @mittalyashu has a similar suggestion.

> Add italics version for light, dimmed and dark theme.

| Example |
| --- |
| <img width="347" alt="CleanShot 2021-09-16 at 20 39 10@2x" src="https://user-images.githubusercontent.com/29014463/133637590-1547760f-a97b-403d-b56e-09cdd7e9ce7e.png"> |

**Comment by mjsarfatti** [comment]
In the meantime if anyone else wants to add italics to any of their themes (including Github theme), you can copy the following snippet in your settings.json: https://gist.github.com/mjsarfatti/ab0c9386436d47b7a7c8c4fae3769670
